frontend/awx/views/schedules/types.ts

```diff
@@ -56,6 +56,7 @@ export interface ScheduleFormWizard {
   launch_config: LaunchConfiguration | null;
   prompt: PromptFormValues;
   schedule_days_to_keep: number;
+  daysToKeepData: number;
```

Comment:
We should decide whether we want `daysToKeepData` or the line above `schedule_days_to_keep`

Reply:
leaning more towards `schedule_days_to_keep`